packages/desktop-client/src/components/manager/subscribe/Login.tsx

```diff
@@ -1,13 +1,16 @@
 // @ts-strict-ignore
-import React, { type ChangeEvent, useState } from 'react';
+import { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
+import { useParams, useSearchParams } from 'react-router-dom';
 
 import { createBudget } from 'loot-core/src/client/actions/budgets';
 import { loggedIn } from 'loot-core/src/client/actions/user';
 import { send } from 'loot-core/src/platform/client/fetch';
 
+import { AnimatedLoading } from '../../../icons/AnimatedLoading';
 import { theme } from '../../../style';
 import { Button, ButtonWithLoading } from '../../common/Button';
+import { ButtonLink } from '../../common/ButtonLink';
```

Comment:
```suggestion
import { ButtonLink } from '../../common/Link';
```

These were consolidated into one file